### ISSUE TITLE
Export All Pattern Stems at Once

### ISSUE THREAD

**Issue by clickysteve** [polyend/TrackerIssues#955]
Hello.

Currently you can't export all pattern stems at once. Only the Pattern which is currently active.

Can we add an option which will export *all* pattern stems in one go please? Thanks

**Comment by bambrose** [comment]
Just bung 'em all in a song and export song stems... as a workaround 
Would be cool just to export all patterns though

**Comment by clickysteve** [comment]
Yeah, that's what I'm doing. It means you have to manually chop etc though, which is a few more steps than is ideal. If you can export one pattern you should be able to export em all!

**Comment by bambrose** [comment]
Makes sense

**Comment by gherkins** [comment]
> export em all!

+1
